frontend/rolecall/src/app/cast/cast-editor-v2.component.ts

```diff
@@ -7,6 +7,13 @@ import { Cast, CastApi } from '../api/cast_api.service';
 import { Piece, PieceApi } from '../api/piece_api.service';
 import { CastDragAndDrop } from './cast-drag-and-drop.component';
 
+type PopupData = {
+  name: string;
+  sortString: string;
+  hasSibling: boolean;
+  pieceIx: number;
```

Comment:
pieceIndex

Reply:
I use Ix for Index all over the place. Does it break a google rule?

Reply:
I noticed :) 

Here's the snippet from our internal requirements for var naming in JavaScript/TypeScript:
![image](https://user-images.githubusercontent.com/6393085/94307316-61eebf80-ff29-11ea-9237-af2f270ce5b0.png)

Reply:
I was hoping that Ix would considered as as valid as Id. Is the difference that Index is shorter than Identifier?

Reply:
I think difference is not about the character length, but more about what's considered "common knowledge" in computer science. "id" is globally known as a shorthand for "identifier", while "ix" is a personal convention not globally considered a standard.

(anecdotally, I've never seen ix used in my ~15yr career, but that's not really a deciding factor, just a data point)